### ISSUE TITLE
High CPU and RAM Load

### ISSUE THREAD

**Issue by Marco674** [meetfranz/plugins#227]
Hi,

I'm the only one that found that Franz is taking a lot of CPU and RAM usage ?

**Comment by Marascon** [comment]
No, it is using a decent amount of RAM for me as well. CPU is manageable tough.

**Comment by LectorRSS** [comment]
Yes, ise a lot resources (Like Chrome)... Franz Make a unique process by service added (Like Chrome) .. Franz its build by Electron.js, and Electron uses Chromium and Node.js to build your app with HTML, CSS & JS..

Its like a WebBrowser with diferents session by cookie on franz tab

**Comment by kuruvar** [comment]
It is not Franz, that is Chrome fault..

**Comment by danielbayley** [comment]
I wonder if there is anything that can be done about performance? Franz is a monster, and sometimes grinds my high-end MacBook Pro to a halt even with 16GB RAM! 😬

**Comment by laxdog** [comment]
I'd been having similar issues and started to look into it more. If you drill down using activity monitor or something similar you should be able to find the 'Helper' that is causing the problems. You can kill the thread from activity monitor, then check through the tabs in Franz to see which one died. 

If it's continually causing problems, the best thing to do then is just disable that service.

It's likely not really an issue with Franz, it's either Chromium as mentioned, or the way the service has been written. 

I also quite often find that just reloading Franz will solve a lot of these problems. (Shift + cmd + R)
